### ISSUE TITLE
Published package contain only umd codes

### ISSUE THREAD

**Issue by e-jigsaw** [progfay/scrapbox-parser#162]
**Describe the bug**
@progfay/scrapbox-parser@4.0.0 is only contain umd codes.

```
Error: Cannot find module '/opt/build/repo/node_modules/@progfay/scrapbox-parser/lib/index.js'. Please verify that the package.json has a valid "main" entry
```

`lib/index.js`, `lib/index.d.ts`, `esm/index.js` not found.

**To Reproduce**
just install

**Expected behavior**
include `lib` and `esm`

**Desktop (please complete the following information):**

- Node.js: 14.2.0

**Comment by progfay** [comment]
This problem is caused by tsbuildinfo.
Before publishing, remove dist directories and rebuild `lib` and `esm` with `tsc`  .
But [incremental compilation](https://devblogs.microsoft.com/typescript/announcing-typescript-3-4-rc) was enable, so dist directory was not created.

I fix to not remove dist before building, and remove tsc cache files before publishing.

**Comment by progfay** [comment]
@e-jigsaw 
[`@progfay/scrapbox-parser@4.0.1`](https://www.npmjs.com/package/@progfay/scrapbox-parser/v/4.0.1) is published for npm.
Please use this version.

Thank you for your contribution!

**Comment by e-jigsaw** [comment]
I've checked 4.0.1. Still broken.

`"main"` points to `./lib/index.js` but actual file is `lib/src/index.js`

**Comment by progfay** [comment]
@e-jigsaw 

I published [`@progfay/scrapbox-parser@4.0.2`](https://www.npmjs.com/package/@progfay/scrapbox-parser/v/4.0.2) and this version is resolved the problem.
I checked with [sample code](https://github.com/progfay/scrapbox-parser#usage) and it works.

**Comment by e-jigsaw** [comment]
Thank you for quick fix!
